prisma/schema.prisma

```diff
@@ -12,13 +12,24 @@ generator client {
   previewFeatures = ["referentialIntegrity"]
 }
 
+model User {
+  id              String    @id @default(uuid())
+  createdAt       DateTime  @default(now())
+  updatedAt       DateTime  @updatedAt
+  name            String    @db.VarChar(255)
+  decks           Deck[]
+}
+
 model Deck {
   id              String    @id @default(uuid())
   createdAt       DateTime  @default(now())
   updatedAt       DateTime  @updatedAt
   type            String    @db.VarChar(255)
   name            String    @db.Text
   cards           Card[]
+  userId          String
+  user            User      @relation(fields: [userId], references: [id], onDelete: Cascade)
+  @@index([userId])
```

Comment:
Added in the user and relation to decks. Nothing special at this point--it's the same relational format as cards to decks.